src/api/middleware/auth/user_or_teacher_authentication_middleware_test.go

```diff
@@ -100,22 +101,29 @@ func Test_Multi_HandlingOfAuthentication_Authorized_SameUser(t *testing.T) {
 }
 
 type MockTeacherService struct{}
-func (s MockTeacherService) GetUser(userId string) (user *interface{}, err error) {
+
```

Comment:
Chequeate esto para mockear cosas sin hardcodear valores https://github.com/stretchr/testify#mock-package

Reply:
Fixed in 60bb7aa258e4e712124db28816d0fd3e5fc50ed1